nasa/harmony
# Harmony 1410 work item update queue

## Jira Issue ID
HARMONY-1410

## Description
Uses a queue for processing work-item updates.

## Local Test Steps
1. Pull this branch
2. Run the bin/start-postgres-localstack script (may need to delete the service/pod for localstack first)
3. List the queues in localstack - you should see two
```
> awslocal sqs list-queues
{
    "QueueUrls": [
        "http://localhost:4566/queue/large-work-item-update-queue",
        "http://localhost:4566/queue/work-item-update-queue"
    ]
}
```
4. Run a few queries against your local instance and verify that they work

## PR Acceptance Checklist
